Sources/GithubAPI.swift

```diff
@@ -58,6 +58,36 @@ public class GithubAPI {
     }
   }
 
+  class func createComment(url: String, comment: String) {
```

Comment:
Add docs here and throughout explaining the params. What is the expected url format?